### PR TITLE
JAMES-3738 Ease overrides of Encryption

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: server/protocols/protocols-library/src/main/java/org/apache/james/protocols/lib/SslConfig.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
+package org.apache.james.protocols.lib;
```
Comment (chibenwa) [inline server/protocols/protocols-library/src/main/java/org/apache/james/protocols/lib/SslConfig.java:1]: Missing license

**Comment by chibenwa** [comment]
Build was :green_apple: , I force pushed to reword the commits messages so that they include the right ticket numbers

**Review by Arsnael — APPROVED** [review]

**Comment by chibenwa** [comment]
Build was :green_apple: , I force pushed to solve a rebase conflict

**Comment by chibenwa** [comment]
Rebased and force pushed to solve conflict

**Pull request merged**